packages/mdctl-docs/lib/parsers/jsdoc/index.js

```diff
@@ -1,30 +1,38 @@
-const { execSync } = require('child_process'),
+const { spawn } = require('child_process'),
       Path = require('path'),
-      jsdoc = Path.join(__dirname, '../../..', 'node_modules', '.bin', 'jsdoc')
+      jsdoc = Path.join(__dirname, '../../../../..', '.bin', 'jsdoc')
```

Comment:
If you use mdctl-docs as module in a separated project will this work as well?

Reply:
It would not. Is that something we need to support?

Reply:
Right now every module works alone, I think each module should work alone so it can be reused by another tools if needed

Reply:
It could be used by web team as well to document framework

Reply:
Sounds good to me! I'll modify it to ensure it can be used standalone.

Reply:
I ended up writing a function which locates `jsdoc` because it could be in multiple potential locations.

```
function locateJsdoc() {

  let path = __dirname,
      jsdoc

  while (path) {
    const jsdocPathPotential = Path.join(path, 'node_modules', '.bin', 'jsdoc')
    if (!jsdoc && Util.isExecutable(jsdocPathPotential)) {
      jsdoc = jsdocPathPotential
    }
    path = path.substr(0, path.lastIndexOf('/'))
  }

  if (!jsdoc) {
    console.log(__dirname)
    throw new Error('Unable to find JSDoc executable')
  }

  return jsdoc
}
```